blog/_posts/2018-02-14-tooling.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jon Pretty
+title: "Towards A Brighter Tooling Future for Scala"
+---
+
+# Towards A Brighter Tooling Future for Scala
+
+Scala has, for its entire existence, been on the back foot with its tooling
+support. This was true when the language first arrived in 2004 as little more
+than a command-line compiler, through 2010 when Typesafe's (now Lightbend's)
+push for commercial adoption took an early focus on improving the Scala IDE for
+Eclipse, through to today, as the tooling experience is very much improved
+thanks in part to the work of JetBrains on IntelliJ, but still lags behind the
+tooling support some other languages enjoy.
+
+People care about language tools, and it says a lot for Scala that its success
+has come without ever having had the best tooling. But a better development
+experience in other languages has always made their grass seem greener, and
```

Comment:
How about removing `in other languages has always made their grass seem greener, and`? This will make the sentence easier to read.